skate-plugin/src/main/resources/messages/skateBundle.properties

```diff
@@ -0,0 +1,5 @@
+skate.configuration.title=Skate Settings
+skate.configuration.choosePath.title=Choose file:
+skate.configuration.choosePath.dialog.title=Choose Changelog File
```

Comment:
In a followup, let's see if these support templating so we can reuse this rather than hardcode the "Changelog" name